src/PsrContainerExtension.php

```diff
@@ -38,7 +41,11 @@ public function load(ContainerBuilder $container, array $config)
     {
         $container->setParameter('roave.behat.psr.container.included.file', $config['container']);
 
-        $loader = new YamlFileLoader($container, new FileLocator(__DIR__));
-        $loader->load('services.yml');
+        $definition = new Definition(ContainerInterface::class, ["%roave.behat.psr.container.included.file%"]);
```

Comment:
I should say there's no requirement for it to be unique - it's perfectly legitimate to have two services that are the same class configured differently.